lib/hiera.rb

```diff
@@ -5,6 +5,7 @@ class Hiera
   require "hiera/version"
   require "hiera/config"
   require "hiera/util"
+  require "hiera/util/win32" if !!File::ALT_SEPARATOR
```

Comment:
Could remove the guard here, yes... though I had put it here intentionally to signal its platform specific (arguably overkill)

`install.rb` doesn't currently require `lib/hiera` ... so I don't think that would help with the `require_relative`, unless I'm missing something?